libsolidity/ast/AST.h

```diff
@@ -518,6 +518,10 @@ class ContractDefinition: public Declaration, public StructurallyDocumented
 
 	bool abstract() const { return m_abstract; }
 
+	ContractDefinition const* superContract(ContractDefinition const& _mostDerivedContract) const;
+	/// @returns the next constructor in the inheritance hierarchy.
+	FunctionDefinition const* nextConstructor(ContractDefinition const& _mostDerivedContract) const;
```

Comment:
@chriseth @ekpyron Hope it is ok like this, if not, can be moved somewhere else. Also, can be static function with both _contract and _mostDerived as parameters
Let me know what do you think

Reply:
It's a much better place to put this!